examples/lib/stories/bridge_libraries/flame_isolate/simple_isolate_example.dart

```diff
@@ -0,0 +1,183 @@
+import 'dart:math';
+
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/game.dart';
+import 'package:flame_isolate/flame_isolate.dart';
+import 'package:flutter/material.dart';
+
+class SimpleIsolateExample extends FlameGame with HasTappables {
+  static const String description = '''
+    This example showcases a simple FlameIsolate example, making easy to 
+    continually run heavy load possible without stutter.
+    
+    Tap the brown square to swap between running heavy load in in an isolate or
+    synchronous.
+    
+    The selected backpressure strategy used for this example is
+    DiscardNewBackPressureStrategy. This strategy discards all new jobs added to
+    the queue if there is already a job in the queue.
+  ''';
+
+  @override
+  void onMount() {
```

Comment:
This should be `onLoad`, otherwise it could add many of the things that are added in here